astroquery/esa/hubble/tests/test_esa_hubble.py

```diff
@@ -194,7 +195,7 @@ def test_cone_search_coords(self):
         assert "Coordinates must be either a string or " \
                "astropy.coordinates" in err.value.args[0]
 
-    def test_query_hst_tap(self):
```

Comment:
could you keep one `query_hst_tap` test, too to test whether the deprecation warning is issued?

Reply:
Added again and being executed.